tests/test_flow.py

```diff
@@ -3448,7 +3489,16 @@ def test_csv_source_with_none_values():
         "2021-04-21 15:56:53.385444",
     ]
     assert termination_result[1].key == "b"
-    assert termination_result[1].body == ["b", True, None, math.nan, math.nan, None]
+    excepted_result = ["b", True, math.nan, math.nan, math.nan, math.nan]
+    for x, y in zip(termination_result[1].body, excepted_result):
```

Comment:
If you do it like this, you need to check the length as well. Otherwise, an empty result will also pass this test.